saleor/graphql/order/filters.py

```diff
@@ -55,13 +55,27 @@ def filter_created_range(qs, _, value):
     return qs
 
 
+def filter_order_search(qs, _, value):
+    order_fields = [
+        "discount_name",
+        "token",
+        "user_email",
+        "user__email",
```

Comment:
I think this one maybe not necessary because we should always set `user_email` anyway.